shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -68,6 +68,58 @@ - (instancetype)initWithConnection:(NSNumber*)connection
 
 #pragma mark -
 
+// The information of a view that has been attached to the engine.
+@interface FlutterEngineViewRecord : NSObject
+- (instancetype)initWithController:(nonnull FlutterViewController*)controller
+                            engine:(nonnull FlutterEngine*)engine
+                  semanticsEnabled:(BOOL)semanticsEnabled;
+
+- (BOOL)semanticsEnabled;
+
+- (void)setSemanticsEnabled:(BOOL)semanticsEnabled;
```

Comment:
This feels a little misleading. Whether semantics is enabled or not is a global setting at the OS level. Within the Flutter app, it feels like we should be modelling this as a single property maybe on FlutterEngine even if we have an accessibility bridge per view.

I kind of wonder if we could actually maintain a single accessibility bridge but have it handle the view/subtree mapping for all the views across the app?

Reply:
It's more like a problem of naming. In the end we need a class to handle or store view-specific a11y information, whether it's called AccessibilityBridge or not. I'm open to better names.

Reply:
But I'll change this setter with a `notifyUpdated` method to follow SSOT.